packages/ide-extension/src/enhancement/enhancements.json

```diff
@@ -20,7 +21,8 @@
                 "exec": {
                     "extensionId": "sapse.sap-ux-application-modeler-extension",
                     "commandId": "sap.ux.environmentcheck.open"
-                }
+                },
+                "platforms": ["BAS"]
```

Comment:
same comment here for platforms and SBAS. Change it to `environment` and `SBAS` respectively.

Reply:
done!